cluster/etcd.go

```diff
@@ -41,11 +42,21 @@ func (c *Cluster) PrepareBackup(ctx context.Context, snapshotPath string) error
 			if backupServer == nil { // start the download server, only one node should have it!
 				if err := services.StartBackupServer(ctx, host, c.PrivateRegistriesMap, backupImage, snapshotPath); err != nil {
 					log.Warnf(ctx, "failed to start backup server on host [%s]: %v", host.Address, err)
+					errors = append(errors, err)
 					continue
 				}
 				backupServer = host
```

Comment:
Should we break here if the backup server succeed to start

Reply:
fixed.